clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -8069,6 +8069,67 @@ static void handleAMDGPUNumVGPRAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
   D->addAttr(::new (S.Context) AMDGPUNumVGPRAttr(S.Context, AL, NumVGPR));
 }
 
+// Returns true if error
```

Comment:
Comment not necessary, just remove it.

Reply:
(1) Updated `ReleaseNotes.rst` (2) Added codegen testcases in `CodeGenCUDA/amdgpu-kernel-attrs.cu` (3) Removed the comment line as suggested.